tests/test_dagmc_query.py

```diff
@@ -152,33 +157,55 @@ def test_calc_tris_per_vert_surf():
     assert(sorted(three_vols_query._vert_data['tri_per_vert']) == [4, 4, 5, 5])
 
 
-def test_calc_tris_per_vert_rootset():
-    """Tests the calc_tris_per_vertfunction given the rootset
+def test_calc_area_triangle_vol():
+    """Tests part of the calc_area_triangle function
     """
     three_vols = df.DagmcFile(test_env['three_vols'])
-    three_vols_query = dq.DagmcQuery(three_vols, meshset=three_vols.root_set)
-    three_vols_query.calc_tris_per_vert()
-    exp_tpv_len = len(three_vols._my_moab_core.get_entities_by_type(
-        three_vols.root_set, types.MBVERTEX))
-    assert(len(three_vols_query._vert_data['tri_per_vert']) == exp_tpv_len)
+    vol = three_vols.entityset_ranges['volumes'][0]
+    three_vols_query = dq.DagmcQuery(three_vols, vol)
+    three_vols_query.calc_area_triangle()
+    np.testing.assert_almost_equal(list(three_vols_query._tri_data['area']), list(np.full(12, 50)))
+
+
+def test_calc_triangle_aspect_ratio_vol():
+    """Tests part of the calc_triangle_aspect_ratio function
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    vol = three_vols.entityset_ranges['volumes'][0]
+    three_vols_query = dq.DagmcQuery(three_vols, vol)
+    three_vols_query.calc_triangle_aspect_ratio()
+    exp = (10*10*10*np.sqrt(2))/(8*5*np.sqrt(2)*5*np.sqrt(2)*(10-5*np.sqrt(2)))
+    np.testing.assert_almost_equal(list(three_vols_query._tri_data['aspect_ratio']), list(np.full(12, exp)))
+
+
+def test_update_tri_data():
+    """Tests the upadte_tri_data function
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    surf = three_vols.entityset_ranges['surfaces'][0]
+    three_vols_query = dq.DagmcQuery(three_vols, surf)
+    three_vols_query.calc_triangle_aspect_ratio()
+    exp_tar = (10*10*10*np.sqrt(2))/(8*5*np.sqrt(2)*5*np.sqrt(2)*(10-5*np.sqrt(2)))
+    np.testing.assert_almost_equal(list(three_vols_query._tri_data['aspect_ratio']), list(np.full(2, exp_tar)))
+    three_vols_query.calc_area_triangle()
+    np.testing.assert_almost_equal(list(three_vols_query._tri_data['area']), list(np.full(2, 50)))
 
 
-def test_calc_tris_per_vert_dimension_incorrect():
-    """Tests the calc_tris_per_vert function given incorrect dimension
+def test_duplicate_tpv():
```

Comment:
thought: should we be testing this warning/duplicate behavior for every single `calc_X` function? Or is that too redundant? I can imaging a case where one calc_X function accidentally has the lines removed and then it doesn't function properly but it is not caught in the tests.